data_steward/retraction/create_deid_map.py

```diff
@@ -142,10 +142,10 @@ def run_queries(queries):
     :param queries: list of queries
     """
     query_job_ids = []
-    for query_dict in queries:
+    for query in queries:
         logging.info('Creating or renaming _deid_map table with query: %s' %
-                     (query_dict['query']))
-        job_results = bq_utils.query(q=query_dict['query'], batch=True)
+                     (query))
```

Comment:
nit: The redundant parenthesis can be removed.
```suggestion
                     query)
```

Reply:
good catch, thank you! I've taken it out!